Sources/MongoSwift/ChangeStream.swift

```diff
@@ -0,0 +1,174 @@
+import mongoc
+
+/// A `ChangeStreamToken` used for wrapping the resume token.
```

Comment:
This docstring could be a little more descriptive. 

e.g. something like:
 "A token used for manually resuming a change stream. Pass this to the `resumeAfter` or `startAfter` fields of `ChangeStreamOptions` to resume or start a change stream where a previous one left off.

See \[the official documentation\](https://docs.mongodb.com/manual/changeStreams/#resume-a-change-stream) for more information on how to use this."